src/scripts/parseCodeSystems.ts

```diff
@@ -0,0 +1,35 @@
+// Code systems source: https://cts.nlm.nih.gov/fhir/metadata?_format=json
```

Comment:
We should prob add this file to `coveragePathIgnorePatterns` in `jest.config.js`